org.firo.firo-qt.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "org.firo.firo-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "command": "firo-qt",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=/run/tor/control.authcookie:ro"
+    ],
+    "modules": [
+        {
+            "name": "firo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/firod",
+                "install -m 0744 -D -t /app/bin/internal bin/firo-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/firo-cli"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/firoorg/firo/releases/download/v0.14.13.1/firo-0.14.13.1-linux64.tar.gz",
```

Comment:
how does this work on aarch64? An aarch64 flatpak is still built.

why not building from source instead?

Reply:
done

Reply:
what's done? A aarch64 package is still build. I suggested building from source as an alternative.